pkgs/by-name/as/astro-language-server/package.nix

```
@@ -0,0 +1,104 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  stdenvNoCC,
+  jq,
+  moreutils,
+  nodePackages,
+  cacert,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "astro-language-server";
+  version = "2.9.0";
+
+  src = fetchFromGitHub {
+    owner = "withastro";
+    repo = "language-tools";
+    rev = "@astrojs/language-server@${finalAttrs.version}";
+    hash = "sha256-OfPe2bGcYWXZHkFc0iCiU3dPxbSsxX+ZVN6Fd7bMf9o=";
+  };
+
+  sourceRoot = "${finalAttrs.src.name}/packages/language-server";
+
+  pnpm-deps = stdenvNoCC.mkDerivation {
```

Could we use instead:

```suggestion
  pnpmDeps = stdenvNoCC.mkDerivation {
```

Like the attribute usually used with `fetchPnpmDeps` in https://github.com/NixOS/nixpkgs/pull/290715 .

---

Updated this derivation